app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -0,0 +1,60 @@
+package ar.com.wolox.android.example.ui.login;
+
+import android.util.Log;
+import android.widget.Button;
+import android.widget.EditText;
+
+import java.util.Objects;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+/**
+ *
+ */
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    private EditText username;
+
+    private EditText password;
+
+    private Button loginButton;
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void init() {
+        this.username = Objects.requireNonNull(getView()).findViewById(R.id.vLoginUsername);
+        this.password = Objects.requireNonNull(getView()).findViewById(R.id.vLoginPassword);
+        this.loginButton = Objects.requireNonNull(getView()).findViewById(R.id.vLoginButton);
+    }
+
+    @Override
+    public void setListeners() {
+        loginButton.setOnClickListener(view -> getPresenter().startLogin(username.getText().toString(), password.getText().toString()));
```

Comment:
If line is too long, try to continue it in the next line. Moreover, Fragment should be a silly class, so it doesn't understand what is happening, so change `startLogin` method to `loginButtonClicked`. Presenter that's more 'inteligent' will know what to do.